File: lista2sala/2.py
n1=float(input("digite o valor da nota 1:"))
n2=float(input("digite o valor da nota 2:"))
n3=float(input("digite o valor da nota 3:"))
m=(n1+n2+n3)/3
if m>=6:
    print ("aluno APROVADO com média:", '%.2f' %m)
else:
    e2=float(input("Digite a nota do exame de recuperação:"))
    nm=(e2+m)/2
    if nm>=5:
        print("aluno APROVADO com média:", '%.2f' %nm)
    else:
        print("aluno REPROVADO com média:", '%.2f' %nm)

File: listas/lista2sala/2.c
#include <stdio.h>
int main(){
    int n1, n2, m, ne, nm;
    printf("digite a nota 1 e 2:");
    scanf("%d%d", &n1, &n2 );
    m=(n1+n2)/2;
    if (m>=6){
        printf("aluno aprovado %d", m);
    }
    else if(m<6){
        printf("entre com a nota de exame:");
        scanf("%d", &ne);
        nm=ne+m;
        if (nm>=5){
            printf("aluno aprovado %d", nm);
        }
        else{
            printf("aluno reprovado %d", nm);
        }
    }
}
